spec/puppet-syntax_spec.rb

```diff
@@ -24,6 +24,11 @@
     expect(PuppetSyntax.future_parser).to eq(true)
   end
 
+  it 'should support app_management setting setting' do
+    PuppetSyntax.app_management = true
```

Comment:
Ditto, it should be reset after.

If you run this spec then manifests_spec.rb in sequence, the `app_management = false (default)` example fails as the default has been changed. It seems to work on Travis CI, possibly by luck as the files run in the opposite order.